spring-integration-core/src/main/java/org/springframework/integration/aggregator/ExpressionEvaluatingReleaseStrategy.java

```diff
@@ -35,7 +35,7 @@ public ExpressionEvaluatingReleaseStrategy(String expression) {
 	 * be boolean).
 	 */
 	public boolean canRelease(MessageGroup messages) {
-		return ((Boolean) process(messages.getMessages())).booleanValue();
+		return (Boolean) process(messages.getMessages());
 	}
```

Comment:
Yeah! Indeed. Conversion error is much better than `CCE`.
But looks like we have to make that setter `final` to honor Sonarcube. Isn't it ?